data/json/monsters.json

```diff
@@ -2473,6 +2473,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_hollow",
+    "name": "zombie hollow",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":20,
+    "aggression":100,
+    "morale":100,
+    "speed":90,
```

Comment:
OK, so these are much faster than a standard zed (75 speed) despite being non-rigid.  I'd expect 'em to move more slowly since they don't have bones, etc.